src/main/resources/Resources/Language/ko_KR.yml

```diff
@@ -0,0 +1,185 @@
+# 한글화 BY mcsim415
+scoreboardAPINotFound: "§c스코어 보드 API 플러그인을 설치해주세요!"
```

Comment:
ScoreboardAPI is the name of the plugin, it seems that no translation is required

Reply:
There's no problem.

Reply:
Ok, got it.